modules/kubernetes-addons/adot-collector-nginx/locals.tf

```diff
@@ -0,0 +1,60 @@
+locals {
+  name                           = "adot-collector"
+  adot_collector_service_account = "adot-collector"
```

Comment:
This needs to be its own namespace and service-account `adot-collector-nginx`. We can consolidate this going forward.

Reply:
Copy